src/main/java/uk/gov/hmcts/darts/arm/service/impl/DetsBatchCleanupArmResponseFilesServiceImpl.java

```diff
@@ -17,16 +17,16 @@ public class DetsBatchCleanupArmResponseFilesServiceImpl extends BatchCleanupArm
 
 
     public DetsBatchCleanupArmResponseFilesServiceImpl(ExternalObjectDirectoryRepository externalObjectDirectoryRepository,
-                                                         ObjectRecordStatusRepository objectRecordStatusRepository,
-                                                         ExternalLocationTypeRepository externalLocationTypeRepository,
-                                                         ArmDataManagementApi armDataManagementApi,
-                                                         UserIdentity userIdentity, ArmBatchCleanupConfiguration batchCleanupConfiguration,
-                                                         ArmDataManagementConfiguration armDataManagementConfiguration, CurrentTimeHelper currentTimeHelper,
-                                                         ArmResponseFileHelper armResponseFileHelper,
-                                                         @Value("${darts.storage.dets.dets-manifest-file-prefix}") String manifestFilePrefix) {
+                                                       ObjectRecordStatusRepository objectRecordStatusRepository,
+                                                       ExternalLocationTypeRepository externalLocationTypeRepository,
+                                                       ArmDataManagementApi armDataManagementApi,
+                                                       UserIdentity userIdentity, ArmBatchCleanupConfiguration batchCleanupConfiguration,
+                                                       ArmDataManagementConfiguration armDataManagementConfiguration, CurrentTimeHelper currentTimeHelper,
+                                                       ArmResponseFileHelper armResponseFileHelper,
+                                                       @Value("${darts.storage.dets.dets-manifest-file-prefix}") String manifestFilePrefix) {
         super(externalObjectDirectoryRepository, objectRecordStatusRepository, externalLocationTypeRepository, armDataManagementApi, userIdentity,
               batchCleanupConfiguration, armDataManagementConfiguration, currentTimeHelper, armResponseFileHelper,
-              manifestFilePrefix
+              manifestFilePrefix, manifestFilePrefix
```

Comment:
Instead of passing the same object twice to hack around the code so you can reuse, it would be better to remove the manifest file prefix into an abstract method so each inherited class can pass the correct value

Reply:
A method feels wrong for this, as we want it to be a constant value which we can not guarantee with a method, but perhaps a overloaded constructor would be better